package.json

```diff
@@ -83,12 +84,12 @@
     "react-element-to-jsx-string": "14.0.1",
     "react-test-renderer": "16.4.2",
     "regenerator-runtime": "0.12.1",
-    "saucelabs": "1.5.0",
-    "selenium-standalone": "6.15.1",
+    "require-context": "1.1.0",
     "semantic-release": "15.9.9",
     "storybook-addon-jsx": "5.4.0",
     "style-loader": "0.22.1",
     "stylelint": "9.5.0",
+    "tape": "4.9.1",
```

Comment:
Why have two testing frameworks?

Reply:
I was lazy and just copy pasted 😆 I'll update to jest